src/context/layers/impact.ts

```diff
@@ -113,38 +127,39 @@ function getBaselineDataForFeature(
   );
 }
 
+const mergeFeaturesByProperty = (
+  baselineFeatures: {}[],
```

Comment:
```suggestion
  baselineFeatures: Array<object>,
```
I think this works better for readability, for me at least. `{}[]` confused me at first haha, I normally use the `[]` after a explicitly defined type (e.g `LayerType[]`). If you agree, make sure to change the line right below this one as well.

of course, typing the object itself, even with something simple would help a ton here!

Reply:
Ya, I agree with both of these. I think in this case, `baselineFeatures` can _probably_ be typed as a `FeatureCollection`, although I haven't tried that.

(Might well work for `aggregateData` too?)

Reply:
aggregate data is just the properties at this point, to keep the api payload smaller